src/AWS.Deploy.Common/UserDeploymentSettings.cs

```diff
@@ -53,6 +54,13 @@ public class UserDeploymentSettings
         /// <param name="node">The current node that is being processed.</param>
         private void TraverseRootToLeaf(JToken node)
         {
+            if (!string.IsNullOrEmpty(node.Path) && node.Type.ToString().Equals("Array"))
```

Comment:
Can we have some unit tests that exercise this code path?

Reply:
Done